primer/brand
# Add landmark to `<Hero>` component

## Summary


This will resolve the [accessibility engineering review for the `<Hero>` component](https://github.com/github/primer/issues/903). A labelled `<section>`element can help users navigate the page, as the label will describe the section's contents. 

## List of notable changes:



- Swaps out `<div>` for `<section>`, ties landmark with heading
- Adds prop `headingLevel` to change heading level (defaults to `h1`)

## Steps to test:



1. Inspect `<section>` region in DevTools
2. Ensure that `<section>` is tied to heading through `aria-labelledby`

## Supporting resources (related issues, external links, etc):

- [Region landmark example](https://www.w3.org/WAI/ARIA/apg/patterns/landmarks/examples/region.html)

## Contributor checklist:


## Reviewer checklist:
